vue/components/organisms/configurator/configurator.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+    <div class="configurator-container">
+        <div class="loader-container" v-bind:style="loaderStyle" v-if="loading">
+            <slot name="loader" v-if="loading">
+                <loader class="loader" v-bind:loader="'ball-scale-multiple'" />
+            </slot>
+        </div>
+        <div class="configurator-wrapper" v-bind:class="{ loading: loading }">
+            <div class="config" ref="configurator" />
+        </div>
+    </div>
+</template>
+
+<style scoped>
+.configurator-container {
+    position: relative;
+}
+
+.configurator-wrapper {
+    position: relative;
+    text-align: center;
+    transition: opacity 0.125s ease-in;
+}
+
+.configurator-wrapper.loading {
+    opacity: 0;
+}
+
+.configurator-wrapper .configurator {
+    cursor: grab;
+}
+
+.configurator-wrapper .configurator[data-frames="1"] {
+    cursor: default;
+}
+
+.configurator-wrapper .configurator.drag {
+    cursor: grabbing;
+}
+
+.configurator-wrapper .configurator.drag[data-frames="1"] {
+    cursor: default;
+}
+
+.configurator-wrapper .configurator.highlight {
+    cursor: pointer;
+}
+
+.configurator-wrapper .configurator.loading,
+.configurator-wrapper .configurator.preloading,
+.configurator-wrapper .configurator:not(.ready) {
+    cursor: progress;
+}
+
+.loader-container {
+    left: calc(50%);
+    position: absolute;
+}
+</style>
+
+<script>
+import { Ripe } from "ripe-sdk";
+import "ripe-sdk/src/css/ripe.css";
+
+/**
+ * The component that contains the RIPE SDK's configurator
+ */
+export const Configurator = {
+    name: "ripe-configurator",
+    props: {
+        /**

```

Comment:
Having these documented here is enough for now I guess, no need to worry about copying over to the README's table. We'll find a way of generating a static documentation page from this.